eppoclient/configurationrequestor.go

```diff
@@ -40,11 +40,15 @@ func (ecr *experimentConfigurationRequestor) GetConfiguration(experimentKey stri
 }
 
 func (ecr *experimentConfigurationRequestor) FetchAndStoreConfigurations() {
-	result := ecr.httpClient.get(RAC_ENDPOINT)
+	result, err := ecr.httpClient.get(RAC_ENDPOINT)
+	if err != nil {
+		fmt.Println("Failed to fetch RAC response", err)
+		return
+	}
```

Comment:
I need to review canonical logging practice in go - I believe it would be better here to just log at the debug level.

Reply:
or perhaps `warn` level since its unexpected/unintended, but handled

Reply:
standard log package in Go does not directly support different logging levels like Warning. You would typically use log.Println or log.Printf for logging purposes and include a warning message manually. To accomplish this we will be using a third-party logging library that supports log levels, such as logrus, zap, or others.

I believe this is something we should move to (see comment from lucas below) to give our customers customization to use their own logger and I've added it to the scope of our upcoming debugging project.